pkg/proto/nbcontract/v1/kubeletconfig.proto

```diff
@@ -9,7 +9,11 @@ message KubeletConfig {
   repeated Taint taints = 3;
   repeated Taint startup_taints = 4;
   KubeletDiskType kubelet_disk_type = 5;
-  FeatureState kubelet_config_file_status = 6; // will remove this as it's only checking if file content is present
+
+  // kubelet_config_file_status specifies if the kubelet_config_file_content should be applied. 
+  FeatureState kubelet_config_file_status = 6;
```

Comment:
Does not have to be now, but let's at some point normalize the use of  `*_status`, `enable_*`, `*_enabled` etc. in field names (and probably avoid `*_status`, as it could be confusing when specifying the desired state)

Reply:
Will do. Thanks.